app/experimenter/static/js/tests/Branch.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import AddonForm from "experimenter/components/AddonForm";
+
+describe("The `AddonForm` component", () => {
+  const setup = () => {
+    const data = {
```

Comment:
i'd love to see these fixtures replaced by factories. here's how i did that in delivery console:
- Generic factory implementation: https://github.com/mozilla/delivery-console/blob/master/src/tests/factory.js
- Shaped object factory implementation: https://github.com/mozilla/delivery-console/blob/master/src/tests/state/actions/index.js#L10
- Factory usage: https://github.com/mozilla/delivery-console/blob/master/src/tests/state/actions/selectors.test.js#L8